### PR TITLE
Coderefactor

### PR DESCRIPTION
**Pull request by Richa-rani02**
Functionalities Added
1. Refactored Ui for responsiveness
2. Added 404 page
3. added empty screens for all routes
4. Added alert on all functionalities

### CODE REVIEW

File: src/Pages/Error/Empty.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            break;
+            case "/explore":
+            {
+                message = "No video available !!"
+            }
+    }
+    return (
+        <div className="empty-page">
+            <div className="empty-img">
+                <img src="../Assets/novideoavaiable.png" className="image-resp"/>
```
Comment (shivanigangadharan) [inline src/Pages/Error/Empty.jsx]: add alt for image

File: src/Pages/Error/Error404.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import "./Error.css";
+import { useNavigate } from "react-router-dom";
+export const Error404=()=>{
+    let navigate=useNavigate();
+    return(
+        <>
+        <div className="error-page flex-col">
+            <div className="error-img">
+                <img src="../Assets/404.png" className="img-responsive" />
```
Comment (shivanigangadharan) [inline src/Pages/Error/Error404.jsx]: alt here too

File: src/components/Header/Header.jsx [hunk 3]
```diff
@@ -7,12 +7,22 @@ import {useNavigate } from "react-router-dom";
...
 let navigate=useNavigate();
 
 const {state,dispatch,setDrawer}=useStateContext();
 
+const logOutHandler=()=>{
+  const toastId = toast.loading("Logging out...");
+        toast.success("You're logged out successfully", {
+            id: toastId,
+        });
+        authDispatch({ type: authActions.LOGOUT });
```
Comment (shivanigangadharan) [inline src/components/Header/Header.jsx:22]: i like how you've defined actions in one place and used neatly here.

**Review by shivanigangadharan — APPROVED** [review]
Code looks good, you've implemented many features and kept functions as compact as possible.

File: src/Pages/Error/Empty.jsx (hunk 1 above)
Comment (Richa-rani02) [reply src/Pages/Error/Empty.jsx]: done

File: src/Pages/Error/Error404.jsx (hunk 2 above)
Comment (Richa-rani02) [reply src/Pages/Error/Error404.jsx]: done

**Review by tanishk15sharma — APPROVED** [review]

**Review by deepak29-git — APPROVED** [review]

**Pull request merged**